examples/mnist_00.py

```diff
@@ -8,18 +7,18 @@
 from torch.optim import Adadelta
 from torch.optim.lr_scheduler import StepLR
 
-###### HYDRA BLOCK ######
+###### HYDRA BLOCK ###### # noqa: E266
 import hydra
 from hydra.core.config_store import ConfigStore
 from dataclasses import dataclass
```

Comment:
I don't think this HYDRA BLOCK this is a good idea.
imports should be sorted normally. We can call out in the text what things are specific to Hydra.
I wouldn't like to see people copying this HYDRA BLOCK as if it's somehow needed.

Reply:
Hmm ok. I was thinking its nice to fence off the code so it is clear what the diffs are. Maybe I can instead annotate it as a 'DIFF' block.